### DISCUSSION TITLE
Final production release of the website

### DISCUSSION THREAD

**Discussion by JeyDi** [PythonBiellaGroup/website#16] [Announcements]
We have completed our PBG website with all the features in the roadmap.

- [x] Multilingual support
- [x] Identify the structure of the website sections
  - [x] Home
  - [x] Meetup (list of all meetups with video, keywords, contents) with tag
  - [x] Wiki and articles section
  - [x] Next events with calendar
  - [x] Supporters section
  - [x] Speakers section
- [x] Learning section
- [x] Quick Feedback
- [x] Cookies request form
- [x] Migrate contents from the old site
- [x] Template folder for contents
- [x] New home page
- [x] New 404 page
- [x] New CSS to improve design

In particular, we have added with this release:
- New home page with the new design
- Custom CSS and templates
- Blog and comments
- 404 custom page

<hr /><em>This discussion was created from the release <a href='https://github.com/PythonBiellaGroup/website/releases/tag/1.0.0'>Final production release of the website</a>.</em>

**Comment by JeyDi** [comment]
Thank you very much @airaghidavide for all the incredible effort 🚀 ❤️

**Reply by airaghidavide** [reply]
Thanks a lot! We had such a great time 🚀
